### DISCUSSION TITLE
Share ALL albums with partner?

### DISCUSSION THREAD

**Discussion by Str1atum** [immich-app/immich#9924] [Q&A]
Is there a possibility to share ALL my albums automatically with my partner which have timeline sharing enabled?
Or do I need to manually add the user to each album?

**Comment by schuhbacca** [comment]
Currently you have to do it manually, I think there are some open feature requests to bulk share or share in the partner feature.

**Comment by trbt** [comment]
I faced the same problem so created a little [script](https://gist.github.com/trbt/8285640be6d6dfa4372dab19cad4fe09) to help with the task.
